### PR TITLE
Improve error messages for file_list arg problems in FileReader

### PR DESCRIPTION
**Pull request by JanuszL**
- adds a path to file_list in error related to it in FileReader

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
- Adds more verbose message - including path to file_list, to FileReader when there is failure caused by missing or malformed file_list

#### What happened in this PR?
 - Adds path tot file_list in the error messages in FileReader

**JIRA TASK**: NA

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: dali/test/python/test_operator_crop_mirror_normalize.py [hunk 1]
```diff
@@ -154,9 +154,9 @@ def crop_mirror_normalize_func(crop_y, crop_x, crop_h, crop_w, mirror_probabilit
         F, H, W, C = image.shape[0], image.shape[1], image.shape[2], image.shape[3]
     assert H >= crop_h and W >= crop_w
 
-    start_y = int(np.round(np.float32(crop_y) * np.float32(H - crop_h)))
+    start_y = int(np.float32(crop_y) * np.float32(H - crop_h) + 0.5)
```
Comment (jantonguirao) [inline dali/test/python/test_operator_crop_mirror_normalize.py]: I think we should do
```suggestion
    start_y = int(np.float32(crop_y) * np.float32(H - crop_h) + np.float32(0.5))
```
I'll submit a patch

File: dali/util/image.cc [hunk 2]
```diff
@@ -83,17 +83,23 @@ void LoadImages(const vector<string> &image_names, ImgSetDescr *imgs) {
...
   std::ifstream file(image_list);
+  int loaded_images = 0;
   if (file.is_open()) {
     std::string filename;
     while (file >> filename) {
       DALI_ENFORCE(!filename.empty());
       std::string full_path = image_folder + "/" + filename;
       if (is_supported_extension(filename, supported_extensions) && !is_empty_file(full_path)) {
         image_names.push_back(full_path);
+        ++loaded_images;
```
Comment (jantonguirao) [inline dali/util/image.cc]: I think we should use `image_names.size()` instead of loaded_images. I'll add it to the patch

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Pull request merged**